integrations/hadoop/documentation.yaml

```diff
@@ -0,0 +1,146 @@
+exporter_type: sidecar
+app_name_short: Hadoop
+app_name: Apache {{app_name_short}}
+app_site_name: Apache Hadoop
+app_site_url: https://hadoop.apache.org/
+exporter_name: the JMX exporter
+exporter_pkg_name: jmx-exporter
+exporter_repo_url: https://github.com/bitnami/containers/tree/main/bitnami/jmx-exporter
+dashboard_available: true
+minimum_exporter_version: "0.17.0"
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+config_mods: |
+  apiVersion: v1
```

Comment:
IIUC, lines 14-22 all need to be added to the existing config yaml. We should make that clear by appending a "+" to these lines.